omnibus-base-mainnet-andromeda.toml

```diff
@@ -15,7 +15,7 @@ include = [
     "tomls/omnibus-base-mainnet-andromeda/perps/referrers.toml",
     "tomls/omnibus-base-mainnet-andromeda/perps/global.toml",
     "tomls/oracles/perps-keeper-cost.toml", # Add gas oracle for keeper fees
-#    "tomls/omnibus-base-goerli-andromeda/perps/feeCollector.toml", ?? does not exist on mainnet
+#    "tomls/omnibus-base-mainnet-andromeda/perps/feeCollector.toml", ?? commented about until SIP-345 released
```

Comment:
feeCollector.toml exists on mainnet but im unsure if its being released until https://github.com/Synthetixio/synthetix-v3/pull/1969 is finalized

Reply:
hey @cavalier-eth yeah won't be released initially but i kept this part as a placeholder and in order to be make the goerli / mainnet toml files as similar to each other as possible